### PR TITLE
RF-1.0.0.3.1 - kobman version & test script

### PR DESCRIPTION
**Pull request by EtricKombat-zz**
- please evaluate and approve LGTM if no corrections are required

### CODE REVIEW

File: src/main/bash/commands/kobman-version.sh [hunk 1]
```diff
@@ -3,71 +3,29 @@
...
-		kobregistery)
-			__kobman_version_kobregistery
-
-		;;
-		"")
-			echo "verify your command &  try again"	
-
-		;;	
-	esac
+	if [ "$environment_type" == "--environment" ]
```
Comment (panickervinod) [inline src/main/bash/commands/kobman-version.sh:27]: How can environment type be equal to --environment? What happens for -env?

File: tests/commands/test-kob-version.sh [hunk 2]
```diff
@@ -1 +1,92 @@
...
+    else
+        __test_kob_cleanup
+        return 1
+    fi
+    __test_kob_cleanup
+
+}
+
+__test_kob_run
+if [[ $status == "true" ]]; then
```
Comment (panickervinod) [inline tests/commands/test-kob-version.sh:88]: The if  block should be inside __test_kob_run, can't be outside it.

File: tests/commands/test-kob-version.sh [hunk 3]
```diff
@@ -1 +1,92 @@
 
+#!/bin/bash
+
+status="true"
```
Comment (panickervinod) [inline tests/commands/test-kob-version.sh:4]: status to be renamed as test_status , also should be set in __test_run as success or failed
refactor test_status as a local variable in __test_run?

File: tests/commands/test-kob-version.sh [hunk 4]
```diff
@@ -1 +1,92 @@
...
+
+function __test_kob_validate
+{
+
+    __kobman_echo_no_colour "Validating...."
+
+    cat tmp.txt | grep -w $(cat $KOBMAN_DIR/var/version.txt)
+    if [[ "$?" != "0" ]]; then
+        __kobman_echo_no_colour "no version details available"
+        status="false"
```
Comment (panickervinod) [inline tests/commands/test-kob-version.sh:53]: status to be renamed as test_status , also should be set in __test_run as success or failed.

File: tests/commands/test-kob-version.sh [hunk 5]
```diff
@@ -1 +1,92 @@
...
+
+function __test_kob_init
+{
+    echo "Checking for kob..."
+    if [[ -d $KOBMAN_DIR ]]; then
+        source $KOBMAN_DIR/src/kobman-utils.sh
+        __kobman_echo_no_colour "kob found"
+        source $KOBMAN_DIR/bin/kobman-init.sh
+    else
+        status="false"
```
Comment (panickervinod) [inline tests/commands/test-kob-version.sh:14]: remove unwanted echo , only need it to check if kob is present , if not install test should fail and exit with the reason.

**Comment by asa1997** [comment]
@EtricKombat  I am closing this pull request for now

**Pull request closed without merging**